packages/demo-html/public/time-code.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Static HTML Demo</title>
+    <style>
+      #wrapper {
+        width: 600px;
+        height: 400px;
+        margin: 0 auto;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>This popup opens in <span id="seconds">5</span> seconds (via embed code)</h1>
+    <p>The popup is opened automatically after the given time has passed.</p>
+    <button
+      style="margin-top: 200px"
+      id="button"
+      data-tf-custom-open="time"
+      data-tf-custom-open-value="5000"
+      data-tf-popup="moe6aa"
+      data-tf-medium="unit-test"
+    >
+      Open the popup manually instead
+    </button>
+
+    <h2>Customize the time</h2>
+    <p>This can be set on "Share" page as well.</p>
+    <form action="" method="get">
+      <label for="time">Milliseconds</label>
+      <input id="time" name="ms" type="number" value="5000" step="1000" min="1000" />
+      <button type="submit">Set</button>
+    </form>
+
+    <script src="../lib/embed-next.js"></script>
+    <script>
+      const { open, close } = window.tf.createPopup("moe6aa", { medium: "unit-test" });
+
+      document.getElementById("button").onclick = function () {
+        const isOpen = !!document.querySelector(".typeform-popup");
+        isOpen ? close() : open();
```

Comment:
don't we have a `toggle` function?

Reply:
🤔  maybe something went wrong on rebasing, the behavioural triggers have their own folders, this is outside. Will fix it